lib/archethic_web/graphql_schema/resolver.ex

```diff
@@ -33,6 +34,56 @@ defmodule ArchethicWeb.GraphQLSchema.Resolver do
     end
   end
 
+  def get_token(address) do
+    t1 = Task.async(fn -> TransactionChain.fetch_genesis_address_remotely(address) end)
+    t2 = Task.async(fn -> get_transaction_content(address) end)
+
+    with {:ok, {:ok, genesis_address}} <- Task.yield(t1),
+         {:ok, {:ok, transaction_content}} <- Task.yield(t2) do
+      {:ok,
+       %{
+         genesis: genesis_address,
+         name: Map.get(transaction_content, "name"),
+         supply: Map.get(transaction_content, "supply"),
+         symbol: Map.get(transaction_content, "symbol"),
+         type: Map.get(transaction_content, "type"),
+         properties: do_reduce_properties(Map.get(transaction_content, "properties"))
+       }}
+    else
+      {:ok, {:error, :network_issue}} ->
+        {:error, "Network issue"}
+
+      {:ok, {:error, :invalid_transaction}} ->
+        {:error, "Transaction is not a token"}
+
+      {:ok, {:error, :transaction_not_found}} ->
+        {:error, "Transaction does not exist!"}
+    end
+  end
+
+  defp get_transaction_content(address) do
+    case Archethic.search_transaction(address) do
+      {:ok, %Transaction{data: %TransactionData{content: content}, type: :token}} ->
+        case Jason.decode(content) do
+          {:ok, map} -> {:ok, map}
+          _ -> {:error, :decode_error}
```

Comment:
I didn't see the `decode_error` in the `with/else` case

Reply:
Added additional cases.